### PR TITLE
Update Portuguese strings

### PR DESCRIPTION
**Pull request by FrioGitHub**

### CODE REVIEW

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -355,5 +355,5 @@
   <string name="msg_swipe_for_files">Deslize à esquerda para @@arquivos@@.</string>
   <string name="msg_swipe_for_output">Deslize à cima para @@logs de compilação@@.</string>
   <string name="idepref_editor_useSoftTabs_title">Usar soft tab</string>
-  <string name="idepref_editor_useSoftTabs_summary">Escolha se deseja usar espaços em vez de caracteres de tabulação (\\t).</string>
```
Comment (deepkrg17) [inline app/src/main/res/values-pt-rBR/strings.xml]: Don't change, '\\' is the escape character for '\' .

**Review by deepkrg17 — CHANGES_REQUESTED** [review]
Don't change. It's an escape character. '\\\\' -> '\\'

**Comment by FrioGitHub** [comment]
> Don't change. It's an escape character. '\\' -> '\'

ok

**Comment by olhotorto** [comment]
Tem telegram? @FrioGitHub

**Comment by FrioGitHub** [comment]
> Tem telegram? @FrioGitHub

tenho, estou no servidor do AndroidIDE, meu user é Frio

**Review by itsaky — APPROVED** [review]

**Comment by itsaky** [comment]
Thanks for your contribution. Sorry for the delay.

**Pull request merged**